### ISSUE TITLE
Indexer Issue + Run as Service Issue + Suggestion Delete \ Remove From DL List

### ISSUE THREAD

**Issue by Krystm** [Radarr/Radarr#288]
So just a few things,

I am having issues with some of the indexers that do work with Sonarr that do not work with Radarr. AltHUB being one of them that I tested.

I am also running this on Windows, Guessing could be the Cookie post I saw earlier, but the ONLY way for me to access is to "Run as Admin". I tried to use NSSM to add a new service, I can see it running in Processes but cannot access the GUI.

Last is a suggestion - Remove a movie after it has been downloaded I have a bunch of different folders, so once a movie is downloaded and renamed I put it in it's appropriate location. So once successful would be nice to either remove it or maybe make it "Finished" so it's at least in history?

Provide a description of the feature request or bug, the more details the better.
When possible include a log!

Please use our [Discord server](https://discord.gg/NWYch8M) for support or longer discussions.

**Comment by galli-leo** [comment]
First off, AltHub not workings is already an existing issue here ;). Secondly, I think the problem here is that, they do not tell us that they support movie search. Each newznab Indexer tells the client which searches they support. Often indexer do not enable movie search, so there is nothing much we can do :/

However, we will add a force option in the future where it tries anyway.

Additionally, Windows has an inherent limitation / security measure where only admin exes and sometimes services can be accessed from outside a network.

Lastyl, I am not quite following your last issue. Radarr renames a finished download and moves it to the correct folder. It should then delete your old file. If you are using torrents however, files are not deleted, so they can still be seeded.

**Comment by danielrw89** [comment]
I am running Radarr using NSSM Successfully without issues. Just FYI (Server 2016 with UAC on)

**Comment by galli-leo** [comment]
@danielrw89 I think it depends on your version.

**Comment by Krystm** [comment]
Update: I did get it to work using NSSM. I in fact, was using NSSM but it was weird it didn't work right the first time, Re-installed the service and was all good. Used and setup NZB Hydra per suggestion on my reddit post so took care of the Indexer issue.

The last one, so here is what I am doing;
I have my movies in separate folders like Kids Movies, Super Hero Movies, Holiday Movies etc. So when movies are downloaded and renamed properly, I then put them in the correct location. I guess the only thing I could maybe do is make different profiles for NZBd for holiday etc and have Radarr move them post processed.

**Comment by galli-leo** [comment]
You could also add different root folders to Radarr and just set them when adding a movie. This way Radarr should automatically move them to the correct folder.

**Comment by Krystm** [comment]
I will give that a shot and check it out, could make automating THAT much easier lol. Question then is, do I have to let Radarr see my whole collection of movies then? Just worried about everything potentially getting renamed / possibly deleted.

**Comment by Krystm** [comment]
Not sure if it was an update but I cannot for the life of me get this to run as a service any more. I tried using  NSSM which I have used for most of my other services. Radarr, Sab, Hydra etc (Running as my admin acct) but this one just will not take. I tried the "install service" and it didn't appear to work either. The only thing I did notice and maybe I missed it but do the files HAVE to be under "Program Files\Radarr" etc? Cause I do not have it there, I have a special folder for all my automation.
